packages/imperative/src/plugins/utilities/PMFConstants.ts

```diff
@@ -87,8 +87,8 @@ export class PMFConstants {
 
     constructor() {
         this.NPM_NAMESPACE = "@brightside";
-        this.CLI_CORE_PKG_NAME = `${this.NPM_NAMESPACE}/core`;
-        this.IMPERATIVE_PKG_NAME = `${this.NPM_NAMESPACE}/imperative`;
+        this.CLI_CORE_PKG_NAME = ImperativeConfig.instance.hostPackageName;
```

Comment:
Just curious, might be a "convenience" thing, but any reason to replicate the constant from imperative config in PMF constants?

Reply:
I wasn't too sure what things might still be using these constants in the PMF. So I went with consistency until we maybe can come back and resolve this as well when doing #87.